x-pack/plugins/viz_editor/public/editor_plugin_registry.ts

```diff
@@ -20,6 +20,7 @@ export interface Suggestion<S extends VisModel = VisModel> {
   visModel: S;
   title: string;
   iconType: IconType;
+  category: string;
```

Comment:
Used for grouping the suggestions (just the chart type for now)

Reply:
Makes sense